frameworks/Swift/vapor/vapor.dockerfile

```diff
@@ -4,7 +4,7 @@ ADD ./ /vapor
 WORKDIR /vapor
 RUN curl -sL -o apt.vapor.sh https://apt.vapor.sh
```

Comment:
we can remove all this apt.vapor stuff. that was for the toolbox which isn't being used anyway. 
only line we should need to compile is `RUN swift build -c release` :)